configuration/concepts/drugs_concepts.csv

```diff
@@ -472,4 +472,18 @@ a5a5aaf1-541e-4ca6-9753-bd4b0e479f81,,,Ciprofloxacin,Ciprofloxacine,Drug,N/A,,,
 bb9c9e33-00db-4f9f-b230-adde3df9b092,,,Primoquine,Primoquine,Drug,N/A,,,
 497457dc-48c1-40f9-8261-0bd9f4bf6b21,,,Dyphenhydramine,Dyphenhydramine,Drug,N/A,,,
 48bcdab5-adb0-4229-aef3-1168a51c2932,,,Eucalyptus & Diphenhydramine & Phenylephrine,Eucalyptus & Diphenhydramine & Phenylephrine,Drug,N/A,,,
-984e4058-d87f-489f-a28d-138c90565666,,,Silver Nitrate,Silver Nitrate,Drug,N/A,,,
\ No newline at end of file
+984e4058-d87f-489f-a28d-138c90565666,,,Silver Nitrate,Silver Nitrate,Drug,N/A,,,
+a546446e-e789-4d02-b02f-c938c6bd373f,,,Ophtalnol,ophtalnol,Drug,N/A,,,
+252159dc-3a8c-4587-801f-762118de992c,,,Belladone & phenobarbital,Belladone & phenobarbital,Drug,N/A,,,
+8f8920be-f144-4fba-b4db-d32940dcea04,,,Desqualinium & lidocaine,Desqualinium & lidocaïne,Drug,N/A,,,
+05130695-8005-4d57-a803-111265f0f4c1,,,Diclofenac & B complex,Diclofénac & complexe B,Drug,N/A,,,
+66c997ab-d252-43e5-94ca-b5925a60fdbb,,,Diclofenac & Methocarbamol & paracetamol,Diclofenac & Methocarbamol & paracetamol,Drug,N/A,,,
+5ac2861d-71e9-4488-bf29-449d555ddfcb,,,Diosmine & Hesperidine,Diosmine & Hesperidine,Drug,N/A,,,
+909ea570-031b-422e-b28d-17b7f492a1b9,,,Genfibrozil,Gemfibrozil,Drug,N/A,,,
+1d47e474-4ae6-4e47-9b69-2ce6ed7399ac,,,Glucosamine & chondroitin,Glucosamine chondroïtine,Drug,N/A,,,
+a30dc1f2-a26b-4bc0-90a4-e7eb7feea473,,,Nystatin & Zinc Oxyde,Nystatine et oxyde de zinc,Drug,N/A,,,
+bcf73538-3f39-4429-a30d-8499609942cd,,,Multivitamin & Iron,Multivitamines et fer,Drug,N/A,,,
+d644e9bd-7ae2-4ef3-b7b7-739816051213,,,Pentoxyphillin,Pentoxyphilline,Drug,N/A,,,
+792871a0-4a52-4e5d-bdd2-4fd49d636227,,,Podophillin,Podophylline,Drug,N/A,,,
+f72393c8-fda0-4107-9b4b-017888be72b2,,,Vitamin A & D, Vitamine A et D,Drug,N/A,,,

```

Comment:
Let's remove this and keep using [this](https://github.com/mekomsolutions/openmrs-config-haiti/pull/145/files#diff-c57c2d669cc45f0e2f0e74e8ddee7861933c6acc1cefe5a53b43bfb1ada816faR463)